iron-overlay-manager.html

```diff
@@ -22,6 +23,8 @@
 
     _backdrops: [],
 
+    _backdropElement: document.createElement('iron-overlay-backdrop'),
```

Comment:
@zinkkrysty tests are failing because of this line. If you do lazy loading on the getter solves it